redis/README.md

```diff
@@ -0,0 +1,175 @@
+# redis-rs
```

Comment:
Instead of adding a copy (which we then have to keep in sync manually) you can use the `package.readme` key in the Cargo metadata to refer to `../README.md`.